CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
   is insufficient to determine that the text is narrative.
 * Upper cased text is lower cased before checking for verbs. This helps avoid some missed verbs.
 * Adds an `Address` element for capturing elements that only contain an address.
+* Alter some of the unit testcases to increase the test coverage.
```

Comment:
actually, we can remove this line: do not need to include testing changes in the CHANGELOG.